script.js

```diff
@@ -12,3 +12,253 @@ mobileLinks.addEventListener('click', () => {
   title.style.display = 'block';
   mobileLinks.classList.toggle('open');
 });
+
+function createLinks(array) {
+  let list = '';
+  array.forEach((item) => {
+    list += `<li> <a  class="is-link" href="#">${item}</a></li>`;
+  });
+  return list;
+}
+// function createLinks(array) {
+//   const list = document.createElement("ul");
+//   array.forEach((item) => {
+//     list.innerHTML = `<li> <a  class="is-link" href="#">${item}</a></li>`;
+//   });
+//   return list;
+// }
+
+/* grid first article */
+const articleOne = {
+  sectionTitle: 'My Recent Works',
+  sectionImage: {
+    src: 'assets/ImgPlaceholder@2x.png',
+    alt: 'section image',
+  },
+  articleOneTitle: 'Multi-Post Stories',
+  itemParagraph: `A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.`,
+  technologyA1: ['css', 'html', 'bootstrap', 'Ruby'],
+  projectBtn: 'See Project',
+};
+
+const sTitle = document.querySelector('.works-title');
+const article1Title = document.querySelector('.item1-description');
+
+const sPopupTitle = document.createElement('H2');
+const titleHr = document.createElement('hr');
+const sTitleTxt = document.createTextNode(`${articleOne.sectionTitle}`);
+sPopupTitle.appendChild(sTitleTxt);
+sTitle.appendChild(sPopupTitle);
+sTitle.appendChild(titleHr);
+
+const imageItem1 = document.createElement('IMG');
+imageItem1.setAttribute('src', articleOne.sectionImage.src);
+imageItem1.setAttribute('alt', articleOne.sectionImage.src);
+document.querySelector('.sectionImg').appendChild(imageItem1);
+const ulLinks = document.querySelector('.button-light');
+
+const a1Title = document.createElement('h3');
+const aTitleTxt = document.createTextNode(`${articleOne.articleOneTitle}`);
+a1Title.appendChild(aTitleTxt);
+article1Title.insertBefore(a1Title, ulLinks);
+
+const a1P = document.createElement('p');
+const a1PTxt = document.createTextNode(`${articleOne.itemParagraph}`);
+a1P.appendChild(a1PTxt);
+article1Title.insertBefore(a1P, ulLinks);
+
+ulLinks.innerHTML = createLinks(articleOne.technologyA1);
+
+const buttonSm = document.querySelector('.button-orange-sm');
+
+const buttonSmall = document.createElement('button');
+const buttonSmallTxt = document.createTextNode(`${articleOne.projectBtn}`);
+buttonSmall.type = 'submit';
+buttonSmall.appendChild(buttonSmallTxt);
+buttonSm.appendChild(buttonSmall);
+buttonSmall.classList.add('is-link', 'button-orange', 'button-sm');
+
+// gridProjects
+const articles = [
+  {
+    title: 'Profesional Art Printing Data',
+    paragraph: `A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard`,
+    technologyArticle: ['html', 'bootstrap', 'Ruby'],
+    prjBtn: 'See Project',
```

Comment:
- [ ] Please kindly include the project image link. This will be more useful when you start adding your own projects to the portfolio.